src/main/app/common/payment-plan/paymentPlan.ts

```diff
@@ -87,4 +87,14 @@ export class PaymentPlan {
     const plural: string = num < 2 ? '' : 's'
     return `${num} ${word}${plural}`
   }
+
+  deserialize (input?: any): PaymentPlan {
```

Comment:
This payment plan class should not been confused with a model class. There should be no need to serialise or deserialise.

Reply:
i have done same mistake couple of times :), i will learn sometime